### PR TITLE
feat: cross account provisioning for encryption keys

### PR DESCRIPTION
**Pull request by scottd018**

### CODE REVIEW

**Review by ricmmartins — COMMENTED** [review]
Just added some minor suggestions

File: content/docs/aro/cross-tenant-encryption-keys/_index.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+ARO_WORKER_SUBNET=${ARO_CLUSTER_NAME}-worker-subnet
+ARO_APP_NAME=${ARO_CLUSTER_NAME}-aro
+
+KV_RESOURCE_GROUP=${ARO_CLUSTER_NAME}-kv-rg
+KV_LOCATION=eastus
+KV_VAULT_NAME=${ARO_CLUSTER_NAME}-vault
+KV_KEY_NAME=${ARO_CLUSTER_NAME}-key
+```
+
+## Use Case: Cross-Tenant Encryption Setup
```
[the remark this answers is not in the document]
Comment (scottd018) [reply content/docs/aro/cross-tenant-encryption-keys/_index.md]: Fixed in https://github.com/rh-mobb/documentation/pull/414/commits/e39cef0cc57239e31f35b201da58f1cf1a742b93

File: content/docs/aro/cross-tenant-encryption-keys/_index.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+There may be situations where you want to create an ARO cluster where the organization 
+has a policy which has a central entity that controls things such as encryption keys or 
+networking components.  This is desirable in large enterprises due to separation of concerns 
+and limiting areas of control for groups to a small scope.  This does present challenges, as 
+those different groups must be able to integrate with one another.  Often times, the integration 
+is difficult, complex, and confusing.  This document serves as a way to clear up some of 
+the confusion by walking you through scenarios for cross-tenancy in ARO.
+
+### Use Cases
```
[the remark this answers is not in the document]
Comment (scottd018) [reply content/docs/aro/cross-tenant-encryption-keys/_index.md]: Fixed in https://github.com/rh-mobb/documentation/pull/414/commits/e39cef0cc57239e31f35b201da58f1cf1a742b93

**Review by ricmmartins — APPROVED** [review]

File: content/docs/aro/cross-tenant-encryption-keys/_index.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+## Cross-Tenant Encryption Setup
+
+This section covers the guide for cross-tenant encryption.
+
+
+### Setup the ARO Tenant
+
+1. Be sure to login the *the tenant that will host your **ARO cluster***:
+
+```
```
Comment (paulczar) [inline content/docs/aro/cross-tenant-encryption-keys/_index.md:80]: tab these out so they appear indented in the rendered markdown so its clear they're part of the ordered task.

Comment (paulczar) [reply content/docs/aro/cross-tenant-encryption-keys/_index.md:80]: also ```bash

**Review by paulczar — CHANGES_REQUESTED** [review]

File: content/docs/aro/cross-tenant-encryption-keys/_index.md (hunk 3 above)
Comment (scottd018) [reply content/docs/aro/cross-tenant-encryption-keys/_index.md]: Fixed in https://github.com/rh-mobb/documentation/pull/414/commits/a6ebd6f903ebbb609d06d188cfdbbba95ee813a1

**Review by paulczar — APPROVED** [review]

**Pull request merged**